config/fxdata/trapdoor.cfg

```diff
@@ -147,6 +147,8 @@ TimeBetweenShots = 0
 SellingValue = 0
 ;Model of the placed trap
 Model = 0
+;Model of trap while recharging. Leave at 0 for no difference.
+RechargeModel = 0
```

Comment:
why are these called Model instead of AnimationId, like the objects, model would be like trap model 1 = boulder, 2 alarm etc
would be a refactor of the og field as well, but would make more sense if it's consistent

Reply:
> why are these called Model instead of AnimationId, like the objects, model would be like trap model 1 = boulder, 2 alarm etc would be a refactor of the og field as well, but would make more sense if it's consistent

There's loads of custom traps already,... but I could refactor it and keep the old field,...

Reply:
yeah doing both would make most sense I think